packages/react/src/components/Slider/Slider-test.js

```diff
@@ -101,8 +122,7 @@ describe('Slider', () => {
       );
     });
 
-    // ArrowRight is not triggering change in slider, possibly due to userEvent bug
-    it.skip('should change the value upon interacting with the slider', async () => {
+    it('should change the value upon interacting with the slider', async () => {
```

Comment:
I worked out that `userEvent` didn't flesh out `KeyboardEvent.which`, which we were relying on in the component. That property is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#obsolete_properties). I updated the component to not rely on `KeyboardEvent.which`. See commit https://github.com/carbon-design-system/carbon/pull/14297/commits/20e5b9dadf44c7e5babde780e336fe42fd020301.

Reply:
@m4olivei will take a look at this today thanks for your patience!

Reply:
Thanks @andreancardona . Looking forward to your feedback.